source-formatter.properties

```diff
@@ -550,8 +550,7 @@
         modules/apps/frontend-,\
         modules/apps/site-initializer/site-initializer-raylife,\
         modules/apps/site-initializer/site-initializer-testray,\
-        modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal,\
-        modules/sdk
```

Comment:
@ling-alan-huang, can we keep `modules/sdk` here for now?

AFAIK, this folder is not compiled during `ant all` and we may introduce issues if we not exclude it.